### DISCUSSION TITLE
Disable all tabs showing in the navigation section

### DISCUSSION THREAD

**Discussion by s-canchi** [squidfunk/mkdocs-material#2257] [Q&A] [answered]
Hi! 

We use MkDocs-Material for our organization website and love it !

I noticed a change in behavior with the latest updates (v6.2.0 onwards) with all the tabs showing up in the left side navigation bar which previously was restricted to only the folders/documents within the top level tab. Is there a way to disable this current behavior with the newer versions. We find pinning the mkdosc-material version to v6.1.7. reverts to desired results but do not want to continue to use the older version in the long run. Here are some screenshots to help visualize the problem. 

Older version (MkDocs-Material v <=6.1.7): 

![Screen Shot 2021-01-29 at 4 35 06 PM](https://user-images.githubusercontent.com/36385305/106340924-3f124600-6250-11eb-898d-534c362c32b2.png)

Newer version (MkDocs-Material v >= 6.2.0):

![Screen Shot 2021-01-29 at 4 36 35 PM](https://user-images.githubusercontent.com/36385305/106340914-36ba0b00-6250-11eb-8dae-be5a220983da.png)

We are using both `navigation.tabs` and `navigation.instant` features.

**Comment by squidfunk** [comment, marked as answer]
Are you using theme (template) overrides? If so, you may need to adjust those, as [tabs navigation was refactored in 6.2](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#fn:1).

**Reply by s-canchi** [reply]
That's good to know. I tried to use the partials from v6.1.7 for nav-item, tabs and tab-items in theme overrides while using v6.2.6 but don't see any change in the left side navigation toc. Is this the right approach ? I am not very familiar with jinja and/or web design.

**Reply by squidfunk** [reply]
Well, if you can verify that this happens _without_ any of your customizations, it's clearly a bug in the theme we should address.

**Reply by s-canchi** [reply]
Thank you for the pointer! After some testing, it seems the problem was including the `nav.html` partial which contained syntax from the older versions. After removing all navigation related partials, I now get the desired behavior with the latest version (v6.2.7). 
![Screen Shot 2021-02-01 at 2 30 02 PM](https://user-images.githubusercontent.com/36385305/106526167-fb128180-6499-11eb-844c-8df32af9dfee.png)
